entities/PropertyManagementService.js

```diff
@@ -0,0 +1,199 @@
+const WealthService = require('./WealthService');
+const { _ } = require('lodash');
+
+/**
+ * Responsibility:
+ *   Defines a model and associated metadata for a property.
+ */
+module.exports = class PropertyManagementService {
+  static getProperties(gameState) {
+    return gameState.config.propertyConfig.properties;
+  }
+
+  static findProperty(gameState, id) {
+    return this.getProperties(gameState).find((prop) => prop.id === id);
+  }
+
+  static getCurrentPlayerBoardProperty(gameState) {
+    const position = gameState.currentPlayer.position;
+    return this.getProperties(gameState).find(
+      (prop) => prop.position === position
+    );
+  }
+
+  static calculateRent(gameState, boardProperty) {
+    let rentAmount = boardProperty.rent;
+    const playerIndex = gameState.players[boardProperty.ownedBy].id;
+
+    if (boardProperty.group === 'Utilities') {
+      const totalRoll =
+        gameState.turnValues.roll[0] + gameState.turnValues.roll[1];
+      const SINGLE_UTILITY_MULTIPLIER = 4;
```

Comment:
Should I pull these out into the static class?

Reply:
These are configuration values, methinks - are they already in the config. ? The static class can provide them via that ?

Reply:
They are not in the configuration. Do you want them to be?